.github/workflows/main.yml

```diff
@@ -0,0 +1,15 @@
+name: Compressed Size
+
+on: [pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: preactjs/compressed-size-action@v2
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+          pattern: "**/dist/**/{index.module.js,index.css}"
```

Comment:
Since all index.css are equal files for now, could we check `./dist/index.css` only?
As far as I understand this should do the trick:
```
pattern: "{**/dist/**/index.module.js,./dist/index.css}"
```

Reply:
Then action output will be just 8 rows instead of 35+.